lib/ReactViews/Credits/DataAttribution/DataAttributionModal.tsx

```diff
@@ -35,17 +47,14 @@ const DataAttributionBox = styled(Box).attrs({
   @media (max-width: ${props => props.theme.mobile}px) {
     width: 100%;
   }
-`;
 
-const AttributionText = styled(Text).attrs(() => ({ medium: true }))`
-  a {
-    color: ${props => props.theme.textDark};
-    text-decoration: underline;
-
-    img {
-      height: 19px;
-      vertical-align: middle;
-    }
+  /* Default cesium bing map logo is white on transparent which is rendered invisible
+     on our modal with white background. This rule forces the background color of the
+      bing imagery to grey so that it is visible.
+  */
+  ${AttributionText} img[title="Bing Imagery"] {
```

Comment:
Making changes as below will produce better result than grey background.
```
  ${AttributionText} img[title="Bing Imagery"] {
    -webkit-filter: invert(1);
    filter: invert(1);
  }
```

Ref According to https://stackoverflow.com/questions/13325798/invert-colors-of-an-image-in-css-or-javascript
Test: http://ci.terria.io/mike-testing/

Reply:
`-webkit-filter` vendor prefixing is not needed. styled-components will add appropriate vendor prefixes for you.

Reply:
Changed to `filter` :heart: